tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/backportIssue.go

```diff
@@ -0,0 +1,160 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	gogithub "github.com/google/go-github/v48/github"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/constants"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/github"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/issueManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/logger"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/repoManager"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/retrier"
+)
+
+const (
+	// Flag Names
+	toplevelIssueIdFlag = "toplevelIssueId"
+	backportVersionFlag = "backportVersions"
+
+	// Other constants
+	issueAutocreatedTemplate   = "Issue was generated by eksGoTools createCveIssue, by:"
+	eksGoOwner                 = "aws"
+	eksGoRepo                  = "eks-distro-build-tooling"
+	eksGoSupportedVersionsPath = "projects/golang/go/MAINTAINED_EOL_VERSIONS"
+)
+
+type backportIssueOptions struct {
+	backportVersions []string
+	toplevelIssueId  int
+}
+
+var bpOptions = &backportIssueOptions{}
+
+func init() {
+	backportCmd.AddCommand(backportIssueCmd)
+	backportIssueCmd.Flags().IntVarP(&bpOptions.toplevelIssueId, toplevelIssueIdFlag, "i", 0, "Issue ID to be backported e.g. 254")
+	backportIssueCmd.Flags().StringSliceVarP(&bpOptions.backportVersions, backportVersionFlag, "b", nil, "to specify versions to backport use this flag. Multiple versions can be specified separated by commas. e.g. <ver>,<ver>,<ver>")
+
+	requiredFlags := []string{
+		toplevelIssueIdFlag,
+	}
+	for _, flag := range requiredFlags {
+		if err := backportIssueCmd.MarkFlagRequired(flag); err != nil {
+			log.Fatalf("failed to mark flag %v as requred: %v", flag, err)
+		}
+	}
+}
+
+var backportIssueCmd = &cobra.Command{
+	Use:   "issue",
+	Short: "Opens backport issues for top level github issue",
+	Long:  "Opens issues to backport top level issue to EKS-Distro supported versions of Golang",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		label := []string{"golang", "security"}
+		issueState := "open"
+
+		retrier := retrier.New(time.Second*380, retrier.WithBackoffFactor(1.5), retrier.WithMaxRetries(15, time.Second*30))
+
+		token, err := github.GetGithubToken()
+		if err != nil {
+			return fmt.Errorf("getting Github PAT from environment at variable %s: %v", github.PersonalAccessTokenEnvVar, err)
+		}
+		githubClient, err := github.NewClient(cmd.Context(), token)
+		if err != nil {
+			return fmt.Errorf("setting up Github client: %v", err)
+		}
+
+		if bpOptions.backportVersions == nil { //When no option was passed to the flag
```

Comment:
let's be more explicit in the clarifying comment about why we are doing this -- e.g. why we get the list of all EOL versions of Go from the repo if none are provided by the user